synfig-core/src/modules/lyr_freetype/lyr_freetype.cpp

```diff
@@ -675,7 +684,18 @@ Layer_Freetype::new_face(const String &newfont)
 	}
 
 #ifdef _WIN32
-	possible_font_directories.push_back("C:\\WINDOWS\\FONTS\\");
+	// All users fonts
+	std::string windir = Glib::getenv("windir");
+	if (windir.empty()) {
+		possible_font_directories.emplace_back("C:\\WINDOWS\\FONTS\\");
+	} else {
+		possible_font_directories.emplace_back(windir + "\\Fonts\\");
+	}
+	// Windows 10 (1809) Added local user fonts installed to C:\Users\%USERNAME%\AppData\Local\Microsoft\Windows\Fonts
+	std::string localdir = Glib::getenv("LOCALAPPDATA");
+	if (!localdir.empty()) {
+		possible_font_directories.emplace_back(localdir + "\\Microsoft\\Windows\\Fonts\\");
+	}
```

Comment:
As the code is duplicated, would it worth to create a simple static (='local', not in the sense of a class method) function near file header?
```c++
#ifdef _WIN32
// Windows 10 (1809) Added local user fonts installed to C:\Users\%USERNAME%\AppData\Local\Microsoft\Windows\Fonts
static std::string get_user_font_folder() {
...
}
#endif 
```

Reply:
Any idea where to put it? As I can see where is no shared headers between `lyr_freetype` and `widget_fontfamily`.

Reply:
I don't know.
I think Synfig library should provide those system and resource paths :P 
Anyway, not important.